cms/api.py

```diff
@@ -136,6 +136,29 @@ def ensure_catalog_page():
         catalog_page.refresh_from_db()
 
 
+def get_enterprise_page_data():
+    return {
+        "title": "Enterprise Page",
+        "slug": ENTERPRISE_PAGE_SLUG,
+        "description": """Deepen your team’s career knowledge and expand their abilities with
+        MIT xPRO’s online courses for professionals. Develop customized learning
+        for your team with bespoke courses and programs on your schedule. Set a
+        standard of knowledge and skills, leading to effective communication among
+        employees and consistency across the enterprise.""",
+        "action_title": "Find out what MIT xPRO can do for your team.",
+        "headings": [
+            {
+                "type": "heading",
+                "value": {
+                    "upper_head": "THE BEST COMPANIES",
+                    "middle_head": "CONNECT WITH",
+                    "bottom_head": "THE BEST MINDS AT MIT",
+                },
+            },
+        ],
+    }
```

Comment:
I am not quite sure about creating this helper for the data. Also, I am not sure what we should do in this case. @arslanashraf7 What's your suggestion?

Reply:
I don't think we need to create this helper method, I believe creating an Enterprise page directly as a class instance should be good. Majorly this is not meant to be a reusable function nor do we need to keep it because it won't have an actual usage.

Here are a couple of options:

**Option 1**

1. If we keep this it  should return an Enterprise page itself 
2. We don't need to provide this much text in the description since this is only going to be used locally
3. For headings, We can provide ready made objects for headings
4. With the above change, It still doesn't need to be called in the ensure_index_pages method. Please take a look at `ensure_home_page_and_site`

**Option 2**

Maybe we don't really need to configure this page as part of configure_wagtail. But this depends upon:

- How hard it is to manually create this page?
- Would the app functionality break if we don't create this?

@asadali145 @shahbaz-shabbir05

Reply:
@arslanashraf7 It's not too difficult to create this page manually. Plus, the app will still work fine even if we don't create this page.

Reply:
@arslanashraf7 Now, the decision is whether to create this page using a different approach (not sure which one yet) or to remove it altogether from the configure_wagtail command. What do you suggest?

Reply:
> Would the app functionality break if we don't create this?

If 404 at `/enterprise` means a breaking thing then yes it will break. However, the link `/enterprise` is not referenced anywhere on the learner side.

> How hard it is to manually create this page?

You just need to create the page in the CMS and add the required data.